### ISSUE TITLE
1.17.1 doesn't work

### ISSUE THREAD

**Issue by beetlejuiceforpresident1** [pronze/SBA#152]
the new BedWars-0.2.19-23 is incompatible with the latest version of SBA, should I try experimental 1.5.6 build on jenkins?

**Comment by ggtylerr** [comment]
The snapshot build isn't for 1.17.1 either. If you want to use it now, you should probably just have a 1.17 server with ViaVersion.

**Comment by DavyJonesSimRacing** [comment]
I don't think that SBA is working with either 1.17 or 1.17.1 yet. Or at least for me the SBA Store and Upgrades Store elements do not appear. Maybe just run a 16.x server for SBA for now or 1.17x without SBA.

**Comment by DavyJonesSimRacing** [comment]
...and then run 1.17.1 in the Minecraft client and for a 1.16x server where SBA should still work install the ViaVersion plugin https://www.spigotmc.org/resources/viaversion.19254/ as the poster above refers to.
